cfpb/owning-a-home
# Update HTTP image references to HTTPS

As part of the preparation for the cf.gov migration to HTTPS, `<img>` tags need to use HTTPS URLs, not HTTP. Leaving them as HTTP will cause mixed-content warnings in the browser, for example as currently on [this page](https://www.consumerfinance.gov/owning-a-home/explore-rates/) on the live site.

This PR adds the `src/static/img/icon_spinner.gif` image from the [hmda-explorer](https://github.com/cfpb/hmda-explorer/blob/master/src/static/img/icon_spinner.gif) repo, which is better included here instead of being referred to at an arbitrary path.

This PR also includes @KimberlyMunoz commit cbd7287 from #691 that fixes up the CHANGELOG.

## Testing

1. Checkout the repository and follow setup instructions in the `README`.
2. Run your cfgov-refresh server in HTTPS mode using `./runserver.sh ssl`.
3. Configure your `config/config.json` with the following content:

 ```json
{
  "rateCheckerAPI": "https://localhost:8000/oah-api/rates/rate-checker",
  "countyAPI": "https://localhost:8000/oah-api/county",
  "mortgageInsuranceAPI": "https://localhost:8000/oah-api/mortgage-insurance"
} 
```

4. Run `grunt watch` to start the owning-a-home application.
5. Open your browser to [https://localhost:8000/owning-a-home/explore-rates/](https://localhost:8000/owning-a-home/explore-rates/), ignoring any certificate errors. In the browser developer console Security tab, note that all resources are loaded securely.

## Review

@virginiacc @higs4281 

## Todos

I've made my changes in a new Unreleased section; should I update the version and tag a new release for these changes? I'm not familiar with the release cycle or other ongoing changes to this repo.

## Notes

The change to `src/static/js/modules/loan-comparison/components/output-alert.js` is not ideal as it would be better to avoid a hardcoded static reference. This React code is currently unused so I didn't make an effort to refactor it to more appropriately pass in the static root at the top level template.
